core/shared/src/main/scala/org/http4s/multipart/MultipartParser.scala

```diff
@@ -815,7 +815,7 @@ object MultipartParser {
       .eval(parseHeaders(headerStream))
       .flatMap(headers => Pull.output1(PartStart(headers): Event))
       .productR(pullPartChunks(delimiterBytes, rest))
-      .flatMap { case rest =>
+      .flatMap { rest =>
```

Comment:
I think we should drop the Nits commit (https://github.com/http4s/http4s/pull/7328/commits/f4ec5885cd2fc0f1bf597b357921a6d1b4e56bd4) and keep this PR focused on the new documentation page.